tests/PSR7/Validators/PathValidatorTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace League\OpenAPIValidation\Tests\PSR7\Validators;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Uri;
+use League\OpenAPIValidation\PSR7\Exception\ValidationFailed;
+use League\OpenAPIValidation\PSR7\ValidatorBuilder;
+use PHPUnit\Framework\TestCase;
+
+class PathValidatorTest extends TestCase
+{
+    public function testItThrowsHelpfulExceptionWhenMissingParams() : void
+    {
+        $this->expectException(ValidationFailed::class);
```

Comment:
In this given example there is no `ValidationFailed` expectation. The given schema is invalid since it's not described in spec. Path parameters at least should have `required: true` as there are in path